spec/rollSpec.js

```diff
@@ -0,0 +1,19 @@
+describe("Roll", function(){
+
+  var roll;
+  beforeEach(function() {
+    roll = new Roll();
+  });
+
+  describe('basic properties', function(){
+
+    it('should start with no pins knocked', function(){
+      expect(roll.pinsknockedtotal).toEqual(0);
+    });
+
+    it('can knock pins down after roll', function(){
+      roll.ball(4)
+      expect(roll.pinsknockedtotal).toEqual(4);
+    });
+  });
```

Comment:
This is a very thin wrapper object around the concept of a number - maybe a bit too much